src/libsyntax/ast.rs

```diff
@@ -524,6 +545,9 @@ impl PartialEq for MetaItemKind {
     }
 }
 
+/// A Block,
```

Comment:
Hm, that probably not really helpful (without the example), but I have no suggestion on how to improve it right now :(